{"name": "test_loging[ssss-11111-Invalid login]", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <allureReport.TestCase object at 0x000001F097C9CCD0>, username = 'ssss', password = '11111'\nexp = 'Invalid login'\n\n    @pytest.mark.parametrize(\"username,password,exp\",[(\"www\",\"www\",\"Invalid login, please try again\"),(\"ssss\",\"11111\",\"Invalid login\"),(\"satwinder@123\",\"Satwinder@123\",\"Satwinder Singh\")])\n    def test_loging(self,username,password,exp):\n        driver=webdriver.Chrome()\n        driver.get(\"https://online.btes.co.in/login/index.php\")\n        driver.find_element(By.XPATH,\"//input[@id='username']\").send_keys(username)\n        driver.find_element(By.XPATH,\"//input[@id='password']\").send_keys(password)\n        driver.find_element(By.XPATH,\"//button[@id='loginbtn']\").click()\n        try:\n            if exp == \"Satwinder Singh\":\n                act = driver.find_element(By.XPATH, \"//*[@class='usertext mr-1']\").text\n            else:\n                act = driver.find_element(By.XPATH, \"//a[text()='Invalid login, please try again']\").text\n>           assert act==exp\nE           AssertionError: assert 'Invalid logi...ase try again' == 'Invalid login'\nE             \nE             - Invalid login\nE             + Invalid login, please try again\n\nallureReport.py:24: AssertionError\n\nDuring handling of the above exception, another exception occurred:\n\nself = <allureReport.TestCase object at 0x000001F097C9CCD0>, username = 'ssss', password = '11111'\nexp = 'Invalid login'\n\n    @pytest.mark.parametrize(\"username,password,exp\",[(\"www\",\"www\",\"Invalid login, please try again\"),(\"ssss\",\"11111\",\"Invalid login\"),(\"satwinder@123\",\"Satwinder@123\",\"Satwinder Singh\")])\n    def test_loging(self,username,password,exp):\n        driver=webdriver.Chrome()\n        driver.get(\"https://online.btes.co.in/login/index.php\")\n        driver.find_element(By.XPATH,\"//input[@id='username']\").send_keys(username)\n        driver.find_element(By.XPATH,\"//input[@id='password']\").send_keys(password)\n        driver.find_element(By.XPATH,\"//button[@id='loginbtn']\").click()\n        try:\n            if exp == \"Satwinder Singh\":\n                act = driver.find_element(By.XPATH, \"//*[@class='usertext mr-1']\").text\n            else:\n                act = driver.find_element(By.XPATH, \"//a[text()='Invalid login, please try again']\").text\n            assert act==exp\n        except:\n            allure.attach(driver.get_screenshot_as_png(), name=\"testlogin\",attachment_type=AttachmentType.PNG)\n>           assert False\nE           assert False\n\nallureReport.py:27: AssertionError"}, "attachments": [{"name": "testlogin", "source": "87433bfc-a201-459a-9b83-19efb10858fd-attachment.png", "type": "image/png"}], "parameters": [{"name": "username", "value": "'ssss'"}, {"name": "password", "value": "'11111'"}, {"name": "exp", "value": "'Invalid login'"}], "start": 1737317415705, "stop": 1737317426446, "uuid": "6f7aade3-0199-42d7-94a0-a5ec89dfcf0a", "historyId": "22480f6764787f7b8a2ceb9f4b83cec9", "testCaseId": "b197cb9646f63edf74f0edeeaeae5595", "fullName": "allureReports.first.allureReport.TestCase#test_loging", "labels": [{"name": "feature", "value": "login"}, {"name": "severity", "value": "minor"}, {"name": "parentSuite", "value": "allureReports.first"}, {"name": "suite", "value": "allureReport"}, {"name": "subSuite", "value": "TestCase"}, {"name": "host", "value": "Satwinder"}, {"name": "thread", "value": "1880-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "allureReports.first.allureReport"}]}